api/routes/storeRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+
+const { storeController } = require('../controllers')
+
+const router = express.Router()
+
+router.get('/', storeController.getAllstores)
```

Comment:
```suggestion
router.get('', storeController.getAllstores)
```
이렇게 되면 해당 엔드포인트는 '/stores/' 와 같은 형태로 끝에 '/'가 붙게되면서 REST하지 못하게 됩니다 수정 부탁드릴게요~